addons/medical/functions/fnc_clearFinished.sqf

```diff
@@ -0,0 +1,34 @@
+/*
+ * Author: Orbis2358
+ * Remove finished actions from list
+ *
+ * Arguments:
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [medic, patient] call ace_medical_fnc_clearFinished
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+params ["_caller", "_target"];
+
+private _joint = _target getVariable [QGVAR(jointTreatment), []];
+private _itemCount = 0;
+
+if !(_joint isEqualTo []) then {
+    {
+        if ((_x select 0) isEqualTo _caller) then {
+            _joint deleteAt _itemCount;
```

Comment:
`_itemCount` should be replaced with the `_forEachIndex` instead.

Reply:
That would make it skip the item after the deleted one.
This should really be using `_joint = _joint select {!(_x select 0) isEqualTo _caller}`

Didn't I already comment about that?
Yeah I did. 
https://github.com/acemod/ACE3/pull/6412#discussion_r198965429
I expected that comment to be applied over all similar occurrences in the other files which is why I didn't repost it for every file. Maybe I should've actually written that down.

Reply:
Yes dedmen did comment that part, but I forgot that there was a similar code,